.github/workflows/tests_lmps_tncuda_python.yml

```diff
@@ -119,6 +119,7 @@ jobs:
           echo ${{ env.VENV_NAME }}/bin >> $GITHUB_PATH
           
           # Adding venv name as an output for subsequent steps to reference if needed
+          source ${{ env.VENV_NAME }}/bin/activate
```

Comment:
Were we not activating the virtual environment before?
How was this playing out in the context of the self-hosted runners?

Reply:
Thank you for clarifying!